spring-boot-project/spring-boot-testcontainers/src/test/java/org/springframework/boot/testcontainers/service/connection/ServiceConnectionContextCustomizerTests.java

```diff
@@ -123,7 +123,7 @@ void equalsAndHashCode() {
 				List.of(new ContainerConnectionSource<>("test", this.origin, PostgreSQLContainer.class, "name",
 						annotation3, () -> container1)));
 		assertThat(t1.hashCode()).isEqualTo(t2.hashCode());
-		assertThat(t1).isEqualTo(t2).isNotEqualTo(t3);
+		assertThat(t1).isEqualTo(t2).isNotEqualTo(t3).isNotEqualTo(t3.hashCode());
```

Comment:
`t1` is a `ServiceConnectionContextCustomizer` and `t3.hashCode()` is an `int`.
Did you want to do
```java
assertThat(t1.hashCode()).isEqualTo(t2.hashCode()).isNotEqualTo(t3.hashCode());
```
in the previous line?

Reply:
@jonatan-ivanov Thanks! It's just a copy and paste error. I fixed it in 6b6d2a4.